templates/post_form.html

```diff
@@ -111,12 +111,15 @@
 			<tr>
 				<th>{% trans %}Flag{% endtrans %}</th>
 				<td>
-					<select name="user_flag" id="user_flag">
+					<select name="user_flag" id="user_flag" style="float:left">
 						<option value="">{% trans %}None{% endtrans %}</option>
 						{% for flag, text in config.user_flags %}
 							<option value="{{ flag }}">{{ text }}</option>
 						{% endfor %}
 					</select>
+					{% if config.flag_preview %}
+					<img name="flag_preview" id="flag_preview" style="vertical-align:middle;">
```

Comment:
consider adding 1-3 pixels to the left of the flag.

Reply:
Added a margin to flag preview.